rpc/master/service_server.go

```diff
@@ -65,12 +70,18 @@ func (s *Server) GetService(serviceID string, svc *service.Service) error {
 }
 
 // GetEvaluatedService returns a service where an evaluation has been executed against all templated properties.
-func (s *Server) GetEvaluatedService(request EvaluateServiceRequest, svc *service.Service) error {
-	result, err := s.f.GetEvaluatedService(s.context(), request.ServiceID, request.InstanceID)
+func (s *Server) GetEvaluatedService(request EvaluateServiceRequest, response *EvaluateServiceResponse) error {
+	svc, err := s.f.GetEvaluatedService(s.context(), request.ServiceID, request.InstanceID)
+	if err != nil {
+		return err
+	}
+
+	tenantID, err := s.f.GetTenantID(s.context(), request.ServiceID)
```

Comment:
I disagree. I think the facade should provide discrete chunks of business logic. This RPC layer is combining results from one or more or those facade calls to return to a client.  I guess I don't see how this is any different from a variety of REST calls to where the web tier makes more than one call to the facade